apps/web/ee/components/support/HelpMenuItem.tsx

```diff
@@ -1,13 +1,213 @@
+            </svg>
+          </button>
+          <button
+            className={classNames(
+              "m-1 items-center justify-center p-1.5 outline outline-1 outline-gray-300 hover:bg-gray-100",
+              rating === "Extremely satisfied" ? "bg-gray-300" : ""
+            )}
+            onClick={() => onRatingClick("Extremely satisfied")}>
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36" width="1.5em" height="1.5em">
+              <path
+                fill="#FFCC4D"
+                d="M36 18c0 9.941-8.059 18-18 18S0 27.941 0 18 8.059 0 18 0s18 8.059 18 18"
+              />
+              <path
+                fill="#664500"
+                d="M18 21c-3.623 0-6.027-.422-9-1-.679-.131-2 0-2 2 0 4 4.595 9 11 9 6.404 0 11-5 11-9 0-2-1.321-2.132-2-2-2.973.578-5.377 1-9 1z"
+              />
+              <path fill="#FFF" d="M9 22s3 1 9 1 9-1 9-1-2 4-9 4-9-4-9-4z" />
+              <path
+                fill="#E95F28"
+                d="M15.682 4.413l-4.542.801L8.8.961C8.542.492 8.012.241 7.488.333c-.527.093-.937.511-1.019 1.039l-.745 4.797-4.542.801c-.535.094-.948.525-1.021 1.064s.211 1.063.703 1.297l4.07 1.932-.748 4.812c-.083.536.189 1.064.673 1.309.179.09.371.133.562.133.327 0 .65-.128.891-.372l3.512-3.561 4.518 2.145c.49.232 1.074.123 1.446-.272.372-.395.446-.984.185-1.459L13.625 9.73l3.165-3.208c.382-.387.469-.977.217-1.459-.254-.482-.793-.743-1.325-.65zm4.636 0l4.542.801L27.2.961c.258-.469.788-.72 1.312-.628.526.093.936.511 1.018 1.039l.745 4.797 4.542.801c.536.094.949.524 1.021 1.063s-.211 1.063-.703 1.297l-4.07 1.932.748 4.812c.083.536-.189 1.064-.673 1.309-.179.09-.371.133-.562.133-.327 0-.65-.128-.891-.372l-3.512-3.561-4.518 2.145c-.49.232-1.074.123-1.446-.272-.372-.395-.446-.984-.185-1.459l2.348-4.267-3.165-3.208c-.382-.387-.469-.977-.217-1.459.255-.482.794-.743 1.326-.65z"
+              />
+            </svg>
+          </button>
+        </div>
+        <div className="my-2 flex justify-end">
+          <Button
+            disabled={disableSubmit}
+            loading={loading}
+            onClick={async () => {
+              if (rating && comment) {
+                await sendFeedback(rating, comment);
+              }
+            }}>
+            {t("submit")}
+          </Button>
+        </div>
+        {successMessage && (
+          <div className="mb-4 bg-green-100 p-4 text-green-700">
+            <p className="font-medium">{t("submitted_feedback")}</p>
+          </div>
+        )}
+        {errorMessage && (
+          <div className="mb-4 flex bg-red-100 p-4 text-sm text-red-700">
+            <div className="flex-shrink-0">
+              <ExclamationIcon className="h-5 w-5" />
+            </div>
+            <div className="ml-3 flex-grow">
+              <p className="font-medium">{t("feedback_error")}</p>
+              <p>{t("please_try_again")}</p>
+            </div>
+          </div>
+        )}
+      </div>
```

Comment:
@Jaibles
![Screenshot from 2022-05-19 12-17-23](https://user-images.githubusercontent.com/65426560/169352593-b49d9f3c-3f3f-450f-9b2f-7c119cd4daa4.png)

Reply:
Maybe check if SEND_FEEDBACK_EMAIL isn't defined, let's not show the Feedback form.

Reply:
@zomars the feedback is also stored in the DB so I would want to keep the form

Reply:
Then let's not make it error if email isn't defined.